reference/promise-types/files.markdown

```diff
@@ -1404,6 +1404,30 @@ comparison or linking operations).
 
 **See also:** [Common Body Attributes][Promise Types and Attributes#Common Body Attributes]
 
+### content
+
+**Description:** Content to write into file
+
+**Type:** `string`
+
+**Allowed input range:** (arbitrary string)
+
+**Example:**
+
+```cf3
+files:
+
+  "/path/to/file"
+
+    content => "Hello world!";
+
+```
+
+**History:** Was introduced in 3.16.0
+
+**Note:** You cannot `content` in combination with the other edit operations
```

Comment:
What does the agent do if you do combine those attributes that are mutually exclusive?

Reply:
@nickanderson That is a good question. I am writing some acceptance tests for this. My idea was that `cf-agent` should log the error message and exit with error. Currently it will print something like:
```
error: A file promise with content attribute cannot have edit operations
```

Reply:
See acceptance tests at https://github.com/cfengine/core/pull/4219